process_sentinel.py

```diff
@@ -362,24 +362,24 @@ def set_default_options(inps, template):
                                      inps_name = template_val.inps_name,
                                      default_value= template_val.default_value)
 
-def call_ssara(custom_template_file, slcDir):
 
-        out_file = os.getcwd() + '/' + 'out_download_ssara' 
-        command = 'download_ssara_rsmas.py ' + custom_template_file 
+def call_ssara(custom_template_file, slcDir):
+        out_file = os.getcwd() + '/' + 'out_{script_name}'
+        command = '{script_name}.py ' + custom_template_file
         #command = '('+command+' | tee '+out_file+'.o) 3>&1 1>&2 2>&3 | tee '+out_file+'.e'  # not used because it only works in bash
-        messageRsmas.log(command)
-        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e' 
-        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slcDir + '; ' +  command + '\"'
-        status = subprocess.Popen(command_ssh, shell=True).wait()
-        print('Exit status from download_ssara_rsmas.py:',status)
-
-        out_file = os.getcwd() + '/' + 'out_download_asfserial' 
-        command = 'download_asfserial_rsmas.py ' + custom_template_file 
-        messageRsmas.log(command)
-        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e' 
-        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slcDir + '; ' +  command + '\"'
-        status = subprocess.Popen(command_ssh, shell=True).wait()
-        print('Exit status from download_asfserial_rsmas.py:',status)
+        command_with_piping_output = '('+command+' > '+out_file+'.o) >& '+out_file+'.e'
+        command_ssh = 'ssh pegasus.ccs.miami.edu \"' \
+                      's.cgood;' \
+                      'cd ' + slcDir + '; ' + \
+                      command_with_piping_output + '\"'
+
+        for script_name in ['download_ssara_rsmas', 'download_asfserial_rsmas']:
+            messageRsmas.log(command.format(script_name= script_name))
+            status = subprocess.Popen(command_ssh.format(script_name= script_name), shell=True).wait()
```

Comment:
`command_ssh` doesn't use `script_name` does it?

Reply:
Agreed. However, I'm unsure of a fix at the moment. I think on a meta-level, we need to think about writing reproducible ways of constructing these strings. I'm thinking about writing a lib for creating some of these standard commands (`ssh`, piping to multiple places, etc).

This fix was just to remove some duplicate code. But yes, you're right. We're still not at a readable state.

Reply:
Ok. I think thats fine for now, but I would not recommend keeping that written that way. I had to look up how `string.format()` works to actually understand what was going on, but that is less than desirable in the future.

Reply:
I would recommend using `string.format` in the future as it's the de facto way for inserting variables into strings. I think this format is pretty standard